recipes-bsp/u-boot/files/edison.env

```diff
@@ -11,7 +11,7 @@ do_dfu_alt_info_ifwi=setenv dfu_alt_info "ifwi${hardware_id} raw 0 8192 mmcpart
 dfu_alt_info_reset=reset ram 0x0 0x0
 
 # Kernel load configuration
-bootargs_console=console=ttyMFD2 earlyprintk=ttyMFD2,keep
+bootargs_console=console=tty1 console=ttyS2,115200n8
```

Comment:
Any reason why we add tty1 too? Before we only had the serial console.

Reply:
No particular reason. I can remove it and only add ttyS2

Reply:
If we add something we need to reason about. If we don't need it let's keep the old behavior.

Reply:
I can remove it and only add ttyS2